src/pages/HomePage/HomePage.js

```diff
@@ -8,6 +9,34 @@ import TextField from "@material-ui/core/TextField";
 import Button from "@material-ui/core/Button";
 
 const HomePage = () => {
+  const [state, setState] = useState({ email: "" });
+
+  const handleChange = (event) => {
+    setState({ email: event.target.value });
+  };
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+
+    // alert message when the user input is empty
+    if (state.email === "") {
+      alert("Please enter your email address. (e.g. example@gmail.com)");
```

Comment:
Very minor thing but i think you dont need the period after address. "Pleaser enter your email address (e.g. example@gmail.com)"